packages/wrangler/src/dev.tsx

```diff
@@ -448,35 +447,29 @@ function useLocalWorker(props: {
   return { inspectorUrl };
 }
 
+export interface DirectorySyncResult {
```

Comment:
The types for `tmp.dirSync` call are wrong, so we redefined it here. I'll send a fix upstream soon.